src/iconsManifest/supportedExtensions.ts

```diff
@@ -3118,6 +3118,16 @@ export const extensions: IFileCollection = {
       extensions: ['doc', 'docx', 'docm', 'dot', 'dotx', 'dotm', 'wll'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'webpack',
```

Comment:
You do realize that the `icon` name does not conform with the file name of the icon file.